smarts/env/gymnasium/wrappers/metrics.py

```diff
@@ -120,16 +110,16 @@ def step(self, action: Dict[str, Any]):
             # Update stored costs.
             self._records[self._scen_name][agent_name].record.costs = costs
 
-            if dones[agent_name]:
+            if info[agent_name]["done"]:
```

Comment:
If `terminated` or `truncated` were dictionaries, they could have been used here instead of `info[agent_name]["done"]`.

Reply:
It has been changed to include both.